src/components/react/GameCardSwiper/GameCard.jsx

```diff
@@ -0,0 +1,75 @@
+import { useState, useEffect } from "react";
+import {
+  Autoplay,
+  Navigation,
+  Pagination,
+  Scrollbar,
+  A11y,
+} from "swiper/modules";
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/css";
+import "swiper/css/navigation";
+import "swiper/css/pagination";
+import "swiper/css/scrollbar";
+import "swiper/css/effect-fade";
+import "./swiperstyles.css";
+
+import configData from "./games-config.json";
+
+export default () => {
+  const [cards, setCards] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const data = configData.games;
+        // Filter games to include only those marked as featured
+        const featuredCards = data
+          .filter(game => game.featured)  // Filter based on the featured flag
+          .sort((a, b) => a.name.localeCompare(b.name));  // Sort alphabetically by name
+        console.log(featuredCards);
+        setCards(featuredCards);
+      } catch (error) {
+        console.error("Error fetching data:", error);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  const cardSlots = cards.map(({ name, description, link }) => (
+    <SwiperSlide key={name}>
+      <a href={`games/${link}`} className="card-link"> 
+        <div className="card gamecard">
+          <img
+            src={`/gifs/games-showcase/${link}-showcase.gif`}
+            alt={`Image for ${name}`}
+          />
+          <h3>{name}</h3>
+          <p>{description}</p>
+        </div>
+      </a>
+    </SwiperSlide>
+  ));
+
+  return (
+    <>
+      <Swiper
+        effect={"fade"}
+        fadeEffect={{
+          crossFade: true,
+        }}
+        grabCursor={true}
+        loop={true}
+        pagination={{ clickable: true }}
+        navigation={true}
+        modules={[Navigation, Pagination, Scrollbar, A11y]}
+        className="mySwiper"
+        slidesPerView={3}
+        spaceBetween={35}
```

Comment:
The swiper doesn't account for the width of the browser. So when I change my browser size the cards get pushed into each other, resized and squished:

![image](https://github.com/user-attachments/assets/2b7a7174-eecb-4458-96ba-f0cab721d1a6)

Is this intended? We could set breakpoints (see here: https://swiperjs.com/swiper-api#param-breakpoints) so that below a certain screen size we only show 1 or 2 cards.

I think changing this may be a decent amount of work anyway, so I think it could be its own task and the changes made in this pull request be approved.

Reply:
> The swiper doesn't account for the width of the browser. So when I change my browser size the cards get pushed into each other, resized and squished:
> 
> ![image](https://private-user-images.githubusercontent.com/24860953/364181707-2b7a7174-eecb-4458-96ba-f0cab721d1a6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyOTE3ODEsIm5iZiI6MTcyNjI5MTQ4MSwicGF0aCI6Ii8yNDg2MDk1My8zNjQxODE3MDctMmI3YTcxNzQtZWVjYi00NDU4LTk2YmEtZjBjYWI3MjFkMWE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE0VDA1MjQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0NGFlZGJlYzBlYzVkZDRhNjZhMjU2YzJmOGZmY2Y4MDNhMjQ0NzFlZDVkYmVmMzE0ZGMxZDMxZDZjMWZhZTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bvoEVCEGepTJOvAqtBWpGDB9dQOYL8-G30LOS0md4bA)
> 
> Is this intended? We could set breakpoints (see here: https://swiperjs.com/swiper-api#param-breakpoints) so that below a certain screen size we only show 1 or 2 cards.
> 
> I think changing this may be a decent amount of work anyway, so I think it could be its own task and the changes made in this pull request be approved.

Had to make a few changes, including the breakpoints:

Added swiperRef using useRef. This was introduced to keep a reference to the Swiper instance, allowing Swiper methods (like update()) to be called.

Introduced a handleResize function. This function listens for window resize events and triggers Swiper's update() method to reinitialise the slides and layout when the window is resized.

Used onSwiper prop. This prop captures the Swiper instance and stores it in swiperRef, so it can be accessed for updating Swiper when needed.

Registered the resize event listener in useEffect. This ensures that the resize handler is set up when the component mounts and removed when it unmounts.

Breakpoints added for different screen sizes:
- 1200px and above: 3 slides are shown.
- 1024px to 1199px: 2 slides are shown.
- 768px to 1023px: 1 slide is shown.
This should also make viewing on mobile devices (both phone and tablet) more pleasing.

Adjusted spaceBetween for each breakpoint. Space between slides was adjusted to ensure appropriate spacing for different screen sizes.

This should be much more pleasing to view now.